patches/chrome-browser-prefs-browser_prefs.cc.patch

```diff
@@ -10,3 +10,11 @@ index 50312cdb0324e3dad18c77c96e5e609bfb259541..ed0ab71e9e08494abb5016bcb4e1fce6
  }
  
  // Register prefs applicable to all profiles.
+@@ -908,6 +909,7 @@ void RegisterProfilePrefs(user_prefs::PrefRegistrySyncable* registry,
+ #endif
+ 
+   RegisterProfilePrefsForMigration(registry);
```

Comment:
do we need a patch here? Can't we just do a chromium_src override for `RegisterProfilePrefs`?

Reply:
that will also override the calling from `RegisterUserProfilePrefs` and `RegisterSigninProfilePrefs` to be `RegisterProfilePrefs_ChromiumImpl` and no one calls our overridden `RegisterProfilePrefs`

Reply:
Also it overrides `extensions::ActivityLog::RegisterProfilePrefs` => `extensions::ActivityLog::RegisterProfilePrefs_ChromiumImpl` and bunch of same cases